plugins/package-managers/pub/src/main/kotlin/utils/PubCacheReader.kt

```diff
@@ -85,13 +85,13 @@ internal class PubCacheReader(flutterHome: File? = null) {
         val path = if (type == "hosted" && url.isNotEmpty()) {
             // Packages with source set to "hosted" and "url" key in description set to "https://pub.dartlang.org".
             // The path should be resolved to "hosted/pub.dartlang.org/packageName-packageVersion".
-            "hosted/${url.replace("https://", "")}/$packageName-$packageVersion"
+            "hosted/${url.replace("https://", "").replace("/","%47")}/$packageName-$packageVersion"
```

Comment:
Please add a space after the comma that separates function arguments.

Reply:
Fixed